region/maridia/inner-pink/Halfie Climb Room.json

```diff
@@ -682,25 +682,72 @@
         "Samus will fall into the door transition, bypassing any lock that may be on the door."
       ]
     },
+    {
+      "link": [1, 3],
+      "name": "Direct G-Mode Morph",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "direct",
+          "morphed": true
+        }
+      },
+      "requires": [
+        "canOffScreenMovement",
+        {"or": [
+          "Gravity",
+          "canInsaneJump",
+          "h_canArtificialMorphPowerBomb"
+        ]},
+        {"or": [
+          "Gravity",
+          "HiJump",
+          "canSunkenTileWideWallClimb",
+          {"and": [
+            "canInsaneJump",
+            "h_canArtificialMorphSpringBall"
+          ]}
+        ]}
+      ],
+      "flashSuitChecked": true,
+      "note": [
+        "Roll through the camera scroll blocks in the morph tunnel to overload PLMs then the speed blocks can be walked through.",
+        "It is also possible to perform a blind foosball. Use the screen movement to be able to tell when Samus is in the morph tunnel and when she makes the jump. A morph turnaround can be helpful.",
```

Comment:
Takes some practice to know what to look for, and then it becomes reasonable.  A morph turnaround is helpful if you can see but I didn't find it useful blind.